common-schema.json

```diff
@@ -3,12 +3,17 @@
   "id": "https://fiware.github.io/dataModels/common-schema.json",
   "title": "Common definitions for FIWARE Harmonized Data Models",
   "definitions": {
+    "EntityIdentiferType": {
+      "type": "string",
+      "minLenght": 1,
+      "maxLength": 256,
+      "pattern": "^[\\w\\-:\\.,;\\(\\)/\\{\\}]+$"
```

Comment:
Two comments:

* Check that "all symbols in the ACCII basic set" minus `[&, ?, /. #]` is equal to the symols added to `\w` in this line
* This regex is aligned with the rules at NGSIv2 spec for ID? Or with the rules at NGSIv2 spec for ID + specific Orion implementation rules (that adds additional forbidden chars to the NGSIv2 spec basic set)?